src/util/Util.js

```diff
@@ -469,7 +469,7 @@ class Util extends null {
       color = (color[0] << 16) + (color[1] << 8) + color[2];
     }
 
-    if (color < 0 || color > 0xffffff) throw new RangeError('COLOR_RANGE');
+    if (color < 0 || color >= 0xffffff) throw new RangeError('COLOR_RANGE');
```

Comment:
The current code throws if `color` is greater than `0xffffff`. You made it throw if it's greater than or equal to `0xffffff`

Reply:
That's totally my bad, no idea how I didn't notice that